readiness-tool/vendors.json

```diff
@@ -0,0 +1,627 @@
+{
+  "apps": {
+    "AD EBiS": {
+      "category": [
+				"Analytics"
+			],
+      "uniqueRegExp": [
```

Comment:
Nit: "searchRegex" or just "regex". `RegExp` typically refers to the JS object rather than the string pattern.